diff --git a/src/util/system.vala b/src/util/system.vala
index a7b2d83..4fa80c9 100644
--- a/src/util/system.vala
+++ b/src/util/system.vala
@@ -12,7 +12,7 @@ int number_of_processors() {
 // Return the directory in which Shotwell is installed, or null if uninstalled.
 File? get_sys_install_dir(File exec_dir) {
     File prefix_dir = File.new_for_path(Resources.PREFIX);
-    return exec_dir.has_prefix(prefix_dir) ? prefix_dir : null;
+    return prefix_dir;
 }
 
 string get_nautilus_install_location() {
